packages/launch/src/focused-launch/plan-details/index.tsx

```diff
@@ -38,7 +38,7 @@ const PlanDetails: React.FunctionComponent = () => {
 	};
 
 	return (
-		<div>
+		<div className="focused-launch-container">
```

Comment:
Bit of a nitpick (and I know that this PR initially tacked only the padding), but it'd be great to uniform markup and structure between Plan and Domain detail views, since we're applying the same view container class

In the Plan details view, the header structure is:

- focused launch container
  - back button wrapper
    - go back button
  - header
    - empty div
       - title
       - subtitle


    
In the Domain details view, the structure is:

- focused launch container
  - header
    - back button wrapper
      - go back button
    - title
    - subtitle